src/public/views/changelist.njk

```diff
@@ -20,12 +20,23 @@
   <div class="row">
     <div class="col-12">
       <table class="table table-bordered table-striped entity-table">
+        {% if adminEntity.listDisplay %}
+          <thead class='thead-dark'>
+            <th>{{ metadata.name }}</th>
+            {% for property in adminEntity.listDisplay %}
+              <th>{{ property }}</th>
+            {% endfor %}
+          </thead>
+        {% endif %}
         {% for entity in entities %}
           <tr>
             <td>
               <a href="{{ 'change' | adminUrl(section, metadata, entity) }}">
                 {{ entity | displayName(metadata) }}
               </a>
+            {% for property in adminEntity.listDisplay %}
+              <td>{{ entity }}</td>
+            {% endfor %}
```

Comment:
You have nested `td`s? That shouldn't happen. You should have the `{{ entity | displayName(metadata) }}` if there's no `displayList`, or the columns if there's one cofigured. The link should be on the first column